FullyNoded/Helpers/Tor/MakeRPCCall.swift

```diff
@@ -183,4 +187,167 @@ class MakeRPCCall {
         }
         
     }
+    
+    func listUnspentUTXOs(completion: @escaping (Result<[UTXO], MakeRPCCallError>) -> Void) {
+        retry(20, task: { completion in
+            self.executeCommand(method: .listunspent, completion: completion)
```

Comment:
you need to pass a `param` as `0` with this command which tells the node to return utxos with 0 confirmations